try: 
    print(x)
#except NameError
   # print("The error is occured")"""
except:
    print("exception")
else:
    print("Nothing went wrong") # will be executed when there is no error in try
finally: 
            name = "Ammu"
            print(name)     #will be executed everytime whether there is error or not in try block 